react-ui/src/pages/analyzeVotes/functions.jsx

```diff
@@ -0,0 +1,22 @@
+export function createTraces(points, groupDetails, plotDetails = {
+  mode: 'markers', type: 'scatter',
+}) {
+  const initial = groupDetails.map((details) => ({
+    x: [], y: [], text: [], hovertemplate: '%{text}', ids: [], ...details, ...plotDetails,
+  }));
+  return points.reduce((traces, point, i) => {
+    traces[point.group].x.push(point.x);
+    traces[point.group].y.push(point.y);
+    traces[point.group].ids.push(point.id || i);
+    traces[point.group].text.push(point.text);
+    return traces;
+  }, initial);
+}
+
+export function roundTwoDecimals(num) {
```

Comment:
Is this function needed, or can we use [toPrecision](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision)?

Reply:
(1.22).toPrecision(2) = 1.2
(0.22).toPrecision(2) = 0.22

This makes the data and container appearance less consistent.